src/DurableTask.Netherite/TransportLayer/EventHubs/BlobBatchReceiver.cs

```diff
@@ -140,10 +140,16 @@ public BlobBatchReceiver(string traceContext, EventHubsTraceHelper traceHelper,
                         BlobManager.AsynchronousStorageReadMaxConcurrency.Release();
                     }
 
-                    TEvent[] result = new TEvent[blobReference.PacketOffsets.Count + 1];
+                    if (ignoreBatch)
+                    {
+                        this.traceHelper.LogWarning("{context} blob {blobName} for batch #{seqno} was not found. Ignoring batch since it must have been already delivered.", this.traceContext, blobClient.Name, seqno);
 
-                    if (!skipEntireBatch)
+                        yield return (eventData, new TEvent[0], seqno, null);
+                    }
```

Comment:
What does this return value represent? Especially confused about this `TEvent[0]`

Reply:
it is an empty batch... no events should be processed